.changeset/true-ties-end.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/turso': patch
+---
+
+Fixed the minimum supported core version to match the LibSQL storage dependency.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)